core/templates/dev/head/pages/exploration_editor/EditorNavigationDirective.js

```diff
@@ -38,10 +38,14 @@ oppia.directive('editorNavigation', [function() {
         $scope.postTutorialHelpPopoverIsShown = false;
 
         $scope.$on('openPostTutorialHelpPopover', function() {
-          $scope.postTutorialHelpPopoverIsShown = true;
-          $timeout(function() {
+          if ($scope.isLargeScreen) {
```

Comment:
this line: `$scope.isLargeScreen = (windowDimensionsService.getWidth() >= 1024);`